corehq/util/log.py

```diff
@@ -320,5 +320,12 @@ def get_traceback_string():
     return f.getvalue()
 
 
-def send_HTML_email(subject, recipient, html_content, *args, **kwargs):
-    return _send_HTML_email(subject, recipient, html_content, *args, **kwargs)
+def send_HTML_email(subject, recipient, html_content, is_conditional_alert=None, *args, **kwargs):
```

Comment:
nit: should `is_conditional_alert` default to `False` instead of `None`?

Reply:
yes it should https://github.com/dimagi/commcare-hq/pull/35511/commits/0db48a5db7ca8316530347e8983168fcae97598a